### DISCUSSION TITLE
Using FluentD with the FluentForward Receiver

### DISCUSSION THREAD

**Discussion by lehuyv** [open-telemetry/opentelemetry-collector-contrib#9363] [General]
Hi,

I'm trying to use FluentD with the Fluent Forward Receiver, but all examples I found was using FluentBit, Here are my fluentd.conf files:

\<source\>
  @type http
  port 9880
  bind 0.0.0.0
\</source\>
\<match **\>
  @type forward
  send_timeout 60s
  recover_wait 10s
  hard_timeout 60s

  \<server\>
    name OTELCollector
    host 0.0.0.0
    port 8006
    weight 60
  \</server\>
\</match\>

Here my otelcoll.yaml:

receivers:
  fluentforward:
    endpoint: 0.0.0.0:8006

processors:
  batch:

exporters:
  logging:
    loglevel: debug

service:
  pipelines:
    logs:
      receivers: [fluentforward]
      processors: [batch]
      exporters: [logging]

I tried a simple curl command:
`curl -X POST -d 'json={"foo":"bar"}' http://localhost:9880/app.log`
but can't see the output in the OTEL Collector console.

Any idea, if this is a supported use case?

Regards,

Huy

**Comment by jpkrohling** [comment]
cc @dmitryax

**Comment by dmitryax** [comment]
Looks like `<server>` is configured incorrectly. Set a correct collector's endpoint instead of `host 0.0.0.0` . If you run both fluentd and collector locally it should be `host 127.0.0.1`

**Reply by lehuyv** [reply]
Thanks for the reply,
Currently, I run them as Docker Containers, under a Docker Engine that was installed on Ubuntu on WSL2 on Windows 10.
I'll try to run everthing localy and keep you posted.
Thanks again,
